.github/workflows/python-package.yml

```diff
@@ -36,7 +36,7 @@ jobs:
             - name: Install dependencies
               run: |
                   python -m pip install --upgrade pip
-                  pip install flake8 tox tox-gh-actions
+                  pip install tox-gh-actions
```

Comment:
Is this maybe related to [#533](https://github.com/plone/bobtemplates.plone/issues/533)?